editor/plugins/script_text_editor.cpp

```diff
@@ -2395,6 +2399,8 @@ void ScriptTextEditor::register_editor() {
 	ED_SHORTCUT("script_text_editor/unfold_all_lines", TTR("Unfold All Lines"), Key::NONE);
 	ED_SHORTCUT("script_text_editor/duplicate_selection", TTR("Duplicate Selection"), KeyModifierMask::SHIFT | KeyModifierMask::CTRL | Key::D);
 	ED_SHORTCUT_OVERRIDE("script_text_editor/duplicate_selection", "macos", KeyModifierMask::SHIFT | KeyModifierMask::META | Key::C);
+	ED_SHORTCUT("script_text_editor/duplicate_lines", TTR("Duplicate Lines"), KeyModifierMask::CMD_OR_CTRL | KeyModifierMask::ALT | Key::DOWN);
+	ED_SHORTCUT_OVERRIDE("script_text_editor/duplicate_lines", "macos", KeyModifierMask::CMD_OR_CTRL | KeyModifierMask::ALT | Key::Z);
```

Comment:
Is this a common macOS shortcut for this purpose?

Reply:
I don't know. Honestly I just put anything there because it would have been a duplicate otherwise

Reply:
Would be good to know what's the expected macOS shortcut for this operation (e.g. in VS Code or Xcode). CC @bruvzg @coppolaemilio

Reply:
I just [looked it up](https://code.visualstudio.com/shortcuts/keyboard-shortcuts-macos.pdf). On VS Code its `shift+cmd+down`

Reply:
That might work. Seems like we use this mapping but only for PC here, and there's a different override for macOS:
```cpp
// core/input/input_map.cpp
        inputs = List<Ref<InputEvent>>();
        inputs.push_back(InputEventKey::create_reference(Key::DOWN | KeyModifierMask::SHIFT | KeyModifierMask::CMD_OR_CTRL));
        default_builtin_cache.insert("ui_text_caret_add_below", inputs);

        inputs = List<Ref<InputEvent>>();
        inputs.push_back(InputEventKey::create_reference(Key::L | KeyModifierMask::SHIFT | KeyModifierMask::CMD_OR_CTRL));
        default_builtin_cache.insert("ui_text_caret_add_below.macos", inputs);
```

Reply:
I changed it on macos to `cmd_or_ctrl+shift+down`